docs/tutorials/goodput.md

```
@@ -0,0 +1,74 @@
+    -m Qwen/Qwen3-0.6B \
+    --endpoint-type chat \
+    --streaming \
+    --url localhost:8000 \
+    --request-count 20 \
+    --goodput "ttft:100 inter_token_latency:3.40"
+```
+<!-- /aiperf-run-vllm-default-openai-endpoint-server -->
+
+Example output:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)